app/models/proposal.rb

```diff
@@ -37,11 +37,10 @@ class Proposal < ActiveRecord::Base
   visitable # Used to track user visit associated with processed proposal
 
   has_many :steps
-  has_many :individual_steps, ->{ individual }, class_name: "Steps::Individual"
   has_many :approval_steps, class_name: "Steps::Approval"
   has_many :purchase_steps, class_name: "Steps::Purchase"
-  has_many :completers, through: :individual_steps, source: :completer
-  has_many :api_tokens, through: :individual_steps
+  has_many :completers, through: :steps, source: :completer
```

Comment:
does this need a scope to `individual` -- the parent (root) step will never have a completer.